2_process/src/process_and_style.R

```diff
@@ -18,9 +18,9 @@ style_data <- function(site_data_annotated){
   mutate(site_data_annotated, station_name = as.factor(station_name))
 }
 
-process_annotate_style <- function(site_data, site_info_file){
-  processed = process_data(site_data)
-  annotated = annotate_data(processed, site_info_file)
+process_annotate_style <- function(site_data_file, site_info, outfile){
+  processed = process_data(site_data_file)
+  annotated = annotate_data(processed, site_info)
```

Comment:
cool, this is the first time i've seen someone keep the inner functions but connect them in a single function. i like it.